v03_pipeline/lib/tasks/reference_data/updated_reference_dataset_query.py

```diff
@@ -0,0 +1,42 @@
+import hail as hl
+import luigi
+
+from luigi_pipeline.lib.hail_tasks import GCSorLocalTarget
+from v03_pipeline.lib.paths import valid_reference_dataset_path
+from v03_pipeline.lib.reference_datasets.reference_dataset import (
+    ReferenceDatasetQuery,
+)
+from v03_pipeline.lib.tasks.base.base_loading_run_params import BaseLoadingRunParams
+from v03_pipeline.lib.tasks.base.base_write import BaseWriteTask
+from v03_pipeline.lib.tasks.reference_data.updated_reference_dataset import (
+    UpdatedReferenceDataset,
+)
+
+
+@luigi.util.inherits(BaseLoadingRunParams)
+class UpdatedReferenceDatasetQuery(BaseWriteTask):
+    reference_dataset_query: ReferenceDatasetQuery
+
+    def requires(self):
+        return self.clone(
+            UpdatedReferenceDataset(
+                reference_dataset=self.reference_dataset_query,
```

Comment:
should this contain a reference to `self. reference_dataset_query.requires()` somehow

Reply:
yep!  I will fix prior to merge.

Reply:
There was also a missing `@property` :/.  We will need tests soon!